addons/web/static/src/core/domain_selector/domain_selector.js

```diff
@@ -174,6 +185,12 @@ export class DomainSelector extends Component {
         this.notifyChanges();
     }
 
+    getDescription(node) {
+        const { pathInfo, operatorInfo, value } = node;
```

Comment:
we could maybe avoid to use the ModelFieldSelector in readonly mode and have leafToString compute the path
description too. This would be useful for search model...